ai_engine_sdk/client.py

```diff
@@ -92,12 +92,14 @@ async def make_api_request(
     }
 
     async with aiohttp.ClientSession() as session:
-        logger.debug(f"\n\n 📤 Request triggered : {method} {api_base_url}{endpoint}")
+        logger.debug(f"\n\n 📤 Request triggered : {
```

Comment:
Please, leave as it was. 
We have max length up to 120 chars so far (standard tends to be 80)

Reply:
probably just an auto-reformat from you tool of your choice. It is in many places, please, check all of them.

Reply:
@XaviPeiro we have back set up and configured on this repo? If not lets add that.

Reply:
Fixed the 120 characters formatting.

Reply:
@tanaygodse Not only in here. Please, notice that there are many changes in the code that are not related to the functionality/fix introduced. Let's keep as clean as possible and avoid those unnecessary changes.

Reply:
> @XaviPeiro we have back set up and configured on this repo? If not lets add that.

Nope, there is not. I am opening an issue.